Issue 16

Khyzad opened the pull request:
Add LICENSE and NOTICE files in root and license elements in POM

<!-- Reviewable:start -->

---

This change is [<img src="https://reviewable.io/review_button.svg" height="35" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/hawaiistatedigitalarchives/pid-webservice/28)

<!-- Reviewable:end -->

Khyzad committed:
Add LICENSE and NOTICE files

Khyzad committed:
Add License elements in Minter and Resolver

Khyzad committed:
Merge branch 'Springboot-feature' into issue-16

dagnir commented:
Please also add license headers to all the source files.

---

Reviewed 4 of 4 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved.

---

_Comments from [Reviewable](https://reviewable.io:443/reviews/hawaiistatedigitalarchives/pid-webservice/28#-:-KM0TT_FpiTh12ubTyLW:-749367991)_

<!-- Sent from Reviewable.io -->

dagnir commented:
Fixes #16

Khyzad committed:
Add licenseheader.txt to root of each project

This is done so that NetBeans knows where to retrieve the license
headers to add to each new file.

Khyzad committed:
Add license headers to each Minter java src file

Khyzad committed:
Edit Resolver's licenseheader to match Minter's

Khyzad committed:
Add license header to Resolve java src files

dagnir commented:
Reviewed 39 of 39 files at r2.
Review status: all files reviewed at latest revision, all discussions resolved.

---

_Comments from [Reviewable](https://reviewable.io:443/reviews/hawaiistatedigitalarchives/pid-webservice/28)_

<!-- Sent from Reviewable.io -->
